map/src/infoblock/components/graph/RoadAttributesGraph.jsx

```diff
@@ -77,84 +99,111 @@ export default function RoadAttributesGraph({ name, data, width }) {
         return type;
     }
 
+    function prepareDistance(value) {
+        if (value?.distance) {
+            if (value.distance > 0.01) {
+                return `${(value.distance / 1000).toFixed(2)} km`;
+            } else {
+                return '< 10 m';
+            }
```

Comment:
value.distance < 0.01 (_in meters?_) = '< 10m' ?

Possibly better to use value.distance < 10